### PR TITLE
Testfix/viewapibackend tests

### PR DESCRIPTION
**Pull request by jykae**
- gitignore node_modules
- fix test 9.1, 9.2, 9.3

### CODE REVIEW

**Comment by jykae** [comment]
@marla-singer @nnn I have weird problem with 9.2, 9.3. I can find element on console but webdriver fails the same xpath. Filling form does not succeed :/

**Comment by jykae** [comment]
@brylie Tried all kinds of wait/sleep I could think of, can't get tests work with feedback modal.

**Comment by jykae** [comment]
@marla-singer Ready for review, finally!

**Comment by marla-singer** [comment]
@jykae Reviewing

File: test/viewapibackend.js [hunk 1]
```diff
@@ -18,63 +18,72 @@ test.describe('View API Backend', function() {
     });
     test.beforeEach(function() {
         CommonUtils.signIn(driver);
-        CommonUtils.fillSignInForm(driver, 'test@test.test', 'testuser');
+        CommonUtils.fillSignInForm(driver, 'robot@test.com', 'robottest');
         CommonUtils.goToDashboard(driver);
```
Comment (marla-singer) [inline test/viewapibackend.js]: We can delete this unuseful step

**Review by marla-singer — COMMENTED** [review]
Test passes successful 3/3 runs
Just clean unuseful line and it's ready to merge

**Comment by marla-singer** [comment]
Merging

**Comment by jykae** [comment]
@marla-singer Hmh, it behaved strangely after I removed that line, before each hook fails. But you merged this already, would probably need some wait or global text xpath finds something else than it should find. Did you try to run?

Adding wait before viewApiDetails fixes issue. I make a small hotfix for that.

**Pull request merged**